renovate.json

```diff
@@ -104,7 +104,7 @@
       "matchDatasources": [
         "docker"
       ],
-      "allowedVersions": "<= 2.4.4",
+      "allowedVersions": "<= 3.0.3",
```

Comment:
Shouldn't we allow for patches? So '< 3.1.0' ?

Reply:
Well, I looked at the PodiumD 4.1 wiki page and there is says explicitly `3.0.3` so I assumed that is what we should aim for. Similar how we treat all other PodiumD dependencies. Do we need to do this differently?

Reply:
I personally don't have limiting these dependencies at all but I realise we need something like this because currently we only build ZAC for PodiumD.

Reply:
As agreed, indeed, we will allow patch versions too as this is also the philosophy of the integration team.